src/schema/schema_create.c

```diff
@@ -985,9 +1101,16 @@ __schema_create(WT_SESSION_IMPL *session, const char *uri, const char *config)
      * back it all out.
      */
     WT_RET(__wt_meta_track_on(session));
-    if (import)
+    if (import) {
         F_SET(session, WT_SESSION_IMPORT);
 
+        if (__wt_config_getones(session, config, "import.metadata_file", &cval) == 0 &&
```

Comment:
Are we allowing the user to specify the name of the file from which they want to import from? I thought that this file was always going to be `WiredTiger.export`.

Reply:
Reading the design I got the impression that the file is specified by the user. Otherwise, it would be a boolean flag. @sueloverso did I get it right?

Reply:
The export side of things will call it `WiredTiger.export` but the user can then copy it to whatever they want so we should allow them to give us the name of the file. For example, they might call it `WiredTiger.export.yesterday` or with a date or whatever.

Reply:
If the user does not provide `metadata_file` we shouldn't look for `WiredTiger.export` file in the home directory. There are two more ways to perform import:
1. Provide all required metadata as a string in `file_metadata`.
2. Provide neither `metadata_file` nor `file_metadata`. In this case, WT will try to recover the required information from the table file.
 
The two approaches described above will fail in tiered storage. At least for now.